functions.py

```diff
@@ -0,0 +1,162 @@
+import csv
+import datetime
+import exceptions_tests
+import os #this is needed for using directory paths and manipulating them
+import sys
+from pip._vendor.distlib.compat import raw_input
+
+class Employee(object):
+
+    def __init__(self, employee_id, name, phone, age):
+        self.employee_id = employee_id
+        self.name = name
+        self.phone = phone
+        self.age = age
+
+class Employees_list(object):
+    def __init__(self, list_of_employees):
+        self.list_of_employees = list_of_employees
+
+    def add_employee(self, new_employee):
+        """
+        This function gets the credentials of a new employee from user and adds it to the employees file.
+        """
+        with open(self.list_of_employees, mode='a') as csvFile:
+            fileout = csv.writer(csvFile, delimiter=',', quoting=csv.QUOTE_ALL)
+            fileout.writerow([new_employee.employee_id, new_employee.name, new_employee.phone, new_employee.age])
+
+    def add_employees_from_file(self):
+        """
+        This function gets from the user a file path that contains the data of the employees and adds it to the employees file
+        only if all the data of all employees is supplied.
+        new_employees = a string.
+        """
+        new_employees_file_path = raw_input("Please add the full file path of the employees you wish to add: ")
+        try:
+            with open(new_employees_file_path) as infile:
+                f = open(new_employees_file_path, 'r+')
+                data = f.read()
+                f.close()
+                f = open("employees.csv", "a")
```

Comment:
Separate the read file and adding file to 2 different functions

Reply:
done :)